profiles/video.jsonld

```diff
@@ -0,0 +1,551 @@
+      "rules": [
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time']",
+          "rule": "included"
+        },
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/progress']",
+          "rule": "included"
+        }
+      ]
+    },
+    {
+      "id": "https://w3id.org/xapi/video/templates#volumechange",
+      "inScheme": "https://w3id.org/xapi/video",
+      "prefLabel": {
+        "en": "Volume Change Statement Template for Video Profile."
+      },
+      "verb": "http://adlnet.gov/expapi/verbs/interacted",
+      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
+      "rules": [
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time']",
+          "rule": "included"
+        },
+        {
+          "location": "$.context.extensions['https://w3id.org/xapi/video/extensions/volume']",
+          "rule": "included"
+        }
+      ]
+    },
+    {
+      "id": "https://w3id.org/xapi/video/templates#screenchange",
+      "inScheme": "https://w3id.org/xapi/video",
+      "prefLabel": {
+        "en": "Screen Change Statement Template for Video Profile."
+      },
+      "verb": "http://adlnet.gov/expapi/verbs/interacted",
+      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
+      "rules": [
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time']",
+          "rule": "included"
+        },
+        {
+          "location": "$.context.extensions['https://w3id.org/xapi/video/extensions/full-screen']",
+          "rule": "included"
+        },
+        {
+          "location": "$.context.extensions['https://w3id.org/xapi/video/extensions/screen-size']",
+          "rule": "included"
+        },
+        {
+          "location": "$.context.extensions['https://w3id.org/xapi/video/extensions/video-playback-size']",
+          "rule": "included"
+        }
+      ]
+    },
+    {
+      "id": "https://w3id.org/xapi/video/templates#terminated",
+      "inScheme": "https://w3id.org/xapi/video",
+      "prefLabel": {
+        "en": "Terminated Statement Template for Video Profile."
+      },
+      "verb": "https://w3id.org/xapi/video/verbs/terminated",
+      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
+      "rules": [
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/time']",
+          "rule": "recommended"
+        },
+        {
+          "location": "$.result.extensions['https://w3id.org/xapi/video/extensions/progress']",
+          "rule": "recommended"
+        }
+      ]
+    },
+    {
+      "id": "https://w3id.org/xapi/video/templates#basic",
+      "name": {
+        "en": "Video Interactions"
+      },
+      "objectActivityType": "https://w3id.org/xapi/video/activity-type/video",
+      "rules": [{
```

Comment:
This doesn't mean what's intended. What this template currently means is, every statement with an object activity type of video can only have one of those three verbs (and since there are other templates above that contradict this, I know it isn't what's intended).

Rules are for restricting the behavior of statements that match the determining properties.